data/en/mbs_0203.json

```diff
@@ -0,0 +1,215 @@
+                            "id": "reporting-period-question",
+                            "title": "Are you able to report for the period from {{exercise.start_date|format_date}} to {{exercise.end_date|format_date}}?"
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "when": [{
+                                        "id": "reporting-period-answer",
+                                        "value": "Yes",
+                                        "condition": "equals"
+                                    }],
+                                    "block": "water-volume-block"
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "block": "reporting-period-block-2"
+                                }
+                            }
+                        ],
+                        "title": "Reporting period"
+                    },
+                    {
+                        "type": "Question",
+                        "id": "reporting-period-block-2",
+                        "title": "Reporting period",
+                        "questions": [{
+                            "type": "DateRange",
+                            "id": "reporting-period-question-2",
+                            "title": "What are the dates of the period that you will be reporting for?",
+                            "answers": [{
+                                    "type": "Date",
+                                    "id": "period-from",
+                                    "label": "From",
+                                    "mandatory": true,
+                                    "q_code": "11"
+                                },
+                                {
+                                    "type": "Date",
+                                    "id": "period-to",
+                                    "label": "To",
+                                    "mandatory": true,
+                                    "q_code": "12"
+                                }
+                            ]
+                        }]
+                    },
+                    {
+                        "type": "Question",
+                        "id": "water-volume-block",
+                        "title": "Volume of potable water that was supplied to customers",
+                        "questions": [{
+                            "id": "water-volume-question",
+                            "type": "General",
+                            "title": "For the period {{ format_conditional_date (answers['period-from'], exercise.start_date)}} to {{ format_conditional_date (answers['period-to'], exercise.end_date)}}, what was the volume of {{respondent.trad_as_or_ru_name}}’s <em>potable water that was supplied to customers</em>, in megalitres?",
```

Comment:
font size isnt right the question is too small a font
![image](https://user-images.githubusercontent.com/29206041/39484853-e87f1104-4d6e-11e8-9aa5-1fb6ce245a1d.png)

Reply:
Looks like that's a little out of date. I corrected that yesterday afternoon.